Doc/using/venv-create.inc

```diff
@@ -132,3 +132,24 @@ detail (typically a script or shell function will be used).
 .. versionadded:: 3.8
    PowerShell activation scripts installed under POSIX for PowerShell Core
    support.
+
+The venv module supports an ini config file in the following locations:
+
+* ``$HOME/.venv.ini`` on Linux/Unix (+ Darwin/Mac OSX) platforms
```

Comment:
`$HOME/.config` is the default value for `$XDG_CONFIG_HOME`; if a Python tool follows the spec it should follow it fully IMO.

Could the file be named `venv.cfg` rather than `.ini`?  Windows ini format is an ill-defined format with many variations, whereas `cfg` is more evocative of `configparser` module.